site/erlang-cookie.xml.inc

```diff
@@ -53,45 +53,42 @@ limitations under the License.
   </p>
 
   <p>
-    On Erlang 20.2 or later versions, the cookie location is the following:
-
+    With Erlang versions starting with 20.2, the cookie file locations are:
     <ul>
       <li>
-        <code>%HOMEDRIVE% + %HOMEPATH%\\.erlang.cookie</code> (usually <code>C:\Users\\<i>Current User</i>\.erlang.cookie</code> for user <code>Current User</code>)
-        if either the <code>HOMEDRIVE</code> or <code>HOMEPATH</code> environment variables are set
+        <code>%HOMEDRIVE%\%HOMEPATH%\.erlang.cookie</code> (usually <code>C:\Users\%USERNAME%\.erlang.cookie</code> for user <code>%USERNAME%</code>) if both the <code>HOMEDRIVE</code> and <code>HOMEPATH</code> environment variables are set
```

Comment:
https://github.com/erlang/otp/blob/9afdaa2a401a0dbe5820df48212d33ec2cdb6144/erts/etc/common/erlexec.c#L1558 suggests there is no backslash between those values. The extra byte is allocated for the null terminator?